src/API/src/models/handlers/blobHandler.ts

```diff
@@ -0,0 +1,22 @@
+import { BlobServiceClient } from '@azure/storage-blob';
+import * as fs from 'fs';
+import config from '../../config/config';
+
+const BLOB_FOLDER = config.get('modelOutputBlobFolder');
+
+export const downloadModelOutput = async (modelOutputName, blobLocation) => {
+  const blobServiceClient = BlobServiceClient.fromConnectionString(
+    config.get('blobStorageConnectionString'),
+  );
+  const containerClient = blobServiceClient.getContainerClient(
+    'vectoratlas-container',
+  );
+
+  const blobClient = containerClient.getBlockBlobClient(blobLocation);
+
+  await blobClient.downloadToFile(`${BLOB_FOLDER}${modelOutputName}.tif`);
```

Comment:
Is it always going to be tifs? The modeul upload allows tifs and shp's

Reply:
No, but I needed to get it to work for tifs first - I think we need to make a new story for shp files. I've built it so it should be relatively simple to make it work for shape files as well.